code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -2484,3 +2494,57 @@
 	damage *= bonus
 	hit_and_run = 1 //reset the hit and run bonus
 	return damage
+
+/mob/living/carbon/Xenomorph/Hivelord/proc/hivelord_sting(mob/living/H)
+
+	if(!check_state())
+		return
+
+	if(world.time < last_hivelord_sting + HIVELORD_STING_COOLDOWN) //Sure, let's use this.
+		to_chat(src, "<span class='xenodanger'>You are not ready to Sting again. It will be ready in [(last_hivelord_sting + HIVELORD_STING_COOLDOWN - world.time) * 0.1] seconds.</span>")
+		return
+
+	if(stagger)
+		to_chat(src, "<span class='xenowarning'>You try to sting but are too disoriented!</span>")
+		return
+
+	if(!H.can_zombie() )
+		to_chat(src, "<span class='xenowarning'>Your sting won't affect this corpse. A larva took the essence, it's too decayed, or simply too fresh.</span>")
+		return
+
+	if(!Adjacent(H))
+		if(world.time > (recent_notice + notice_delay)) //anti-notice spam
+			to_chat(src, "<span class='xenowarning'>You can't reach this target!</span>")
+			recent_notice = world.time //anti-notice spam
+		return
+
+	if(!check_plasma(150))
+		return
+	last_hivelord_sting = world.time
+	use_plasma(150)
+
+	addtimer(CALLBACK(src, .hivelord_sting_cooldown), HIVELORD_STING_COOLDOWN)
+
+	recurring_injection(H, "necrovirus")
+
+/mob/living/carbon/Xenomorph/Hivelord/proc/hivelord_sting_cooldown()
+	playsound(loc, 'sound/voice/alien_drool1.ogg', 50, 1)
+	to_chat(src, "<span class='xenodanger'>You feel your virus glands refil, another body can be prepared for reanimation.</span>")
+	update_action_button_icons()
+
+/mob/proc/can_zombie()
+	return FALSE
+
+/mob/living/carbon/monkey/can_zombie()
+	return FALSE
+
+/mob/living/carbon/human/can_zombie()
+	if(stat == DEAD && timeofdeath  && !check_tod() && !chestburst)
```

Comment:
what happened to the flags?

Reply:
Rohesie didn't want a flag in case we wanted to use roundstart corpse spawn for some other purposes.

Reply:
So round start corpses are all able to be necromorph

Reply:
Nope, it checks if timeofdeath is filled in, since all roundstart corpses have their timeofdeath reset and set to 0, they can't be reanimated ever.